blocks/text-media/text-media.css

```diff
@@ -0,0 +1,19 @@
+div.text-media.block {
+    display: block !important;
```

Comment:
@sdmcraft I had to add the `!important` because otherwise this line would be shadowed by https://github.com/hlxsites/zeiss/blob/main/styles/simplified-styles.css#L111
would you know a cleaner way to achieve this? I could also add `:not(.text-media)` to that condition, not sure if thats better.

Reply:
Is keeping the display as `grid` problematic for this block? I found the `grid` based layout useful as it help in reusing the styles and maintaining  the responsive layout behavior exactly as the main site which otherwise we'd have to re-implement and may not be able to match exactly.